### DISCUSSION TITLE
AMQP (RabbitMQ) connection

### DISCUSSION THREAD

**Discussion by alq-eium** [frangoteam/FUXA#1205] [Q&A]
Is it possible to establish a connection between FUXA and an AMQP broker, for instance RabbitMQ? If it is not possible, maybe this question could be transfered to "Ideas" category.

Thank you in advance!

**Comment by unocelli** [comment]
Hi, I suppose it is possible, FUXA uses MQTT.js library

**Reply by alq-eium** [reply]
Thank you @unocelli, sorry for my late reply.
